app/src/views/OperationalLearning/OperationalLearningMap/i18n.json

```
@@ -0,0 +1,8 @@
+{
+    "namespace": "operationalLearning",
+    "strings": {
+        "downloadMapTitle": "Operational learning map",
+        "learningLegendLabel": "Learnings",
+        "learningCountLegendLabel":"Learning count"
```

Let's fix the spacings.
```suggestion
        "learningCountLegendLabel": "Learning count"
```